packages/cli/src/commands/link/linkDependency.js

```diff
@@ -34,8 +34,10 @@ const linkDependency = async (
     }
 
     const isInstalled = linkConfig.isInstalled(
+      // $FlowFixMe
       projectConfig,
```

Comment:
couldn't get Flow to understand this (don't say TS gonna fix it, it probably won't). Help appreciated on getting this solved, but I'm also cool with leaving this as is.

Reply:
It will! Flow has issues understanding whether it's Android/iOS its dealing right now. Run into it in other places too, so good to ignore.

Reply:
Maybe, but that has noting to do with platforms, just figuring out alternatives of types. E.g. If we rewritten this code to switch statement with a lot of repetition, Flow would understand the branching logic, but that would be too much

Reply:
Haha, yeah, I know it has nothing to do with platforms :) Was just an example I used to explain what you said!